### DISCUSSION TITLE
Wolai代码块渲染疑问

### DISCUSSION THREAD

**Discussion by zpoyqx** [F9y4ng/GreasyFork-Scripts#202] [问答专区(Question / Answer)]
我在使用鸿蒙黑体的情况下，代码块里的英文字母会挤在一起，如下图所示：
![后](https://user-images.githubusercontent.com/56358596/230932633-efad7a6f-c05e-4c9c-97ca-d478c267cfd6.png)
但是，我比较喜欢未渲染前的这种代码样式：
![前](https://user-images.githubusercontent.com/56358596/230932724-dd809702-bb8d-459f-9ca5-b1d5693e8521.png)
请问我需要怎么做才能避免渲染代码块，或者推荐我一款字体在中文和英文（类似编程字体）显示都比较好？

**Comment by F9y4ng** [comment]
在**需要渲染的网页元素**中，追加`:not(._2TL9K,._2TL9K *)`，保存为站点独享。
